src/Parser.php

```diff
@@ -14,8 +14,13 @@ public function parse(string $filename)
         foreach ($lines as $line) {
             $line = trim($line);
             if ($line !== '') {
-                [$url, $status] = preg_split('/[\s]+/', $line);
-                $domains[] = new Domain($url, $status);
+                [$url, $status, $content] = preg_split('/[\s]+/', $line);
```

Comment:
That would be reasonable (also, easier for adding new fields later), although it does do away with the insanely simple setup at the moment... It's a tradeoff I guess!